tests/hypervisor/test_default.py

```diff
@@ -112,10 +112,13 @@ def test_associated_info_by_rhsmlog_and_webui(
 
         # check host-to-guest association in rhsm.log
         mappings = result["mappings"]
-        associated_hypervisor_in_mapping = mappings[default_org][guest_uuid][
-            "guest_hypervisor"
-        ]
-        assert associated_hypervisor_in_mapping == host_name
+        if HYPERVISOR == "local":
```

Comment:
We can skip this step for local mode, which already is tested in other test cases.

Reply:
updated